From 6cdbf3706350c4d724ea8d35267ae93a4d99eab0 Mon Sep 17 00:00:00 2001
From: Andreas Henriksson <andreas@fatal.se>
Date: Sat, 14 Mar 2009 17:37:35 +0100
Subject: [PATCH 06/23] iproute2: drop equalize support.

Hello Stephen and netdev people!

Currently you can configure "equalize" and it looks all fine and dandy.
The kernel has the interface defined, but apparently there's never actually
been any implementation for it (only a never merged patch in the 2.4 era).

I'm suggesting to drop the code to give any potential users of this feature
the benefit of receiving a proper error message. I see it unlikely that
this will be implemented in the near future, but if it ever happens
reviving the iproute2 side should be as easy as git revert this patch.

For more details see http://bugs.debian.org/149897

Regards,
Andreas Henriksson
---
 ip/ipmonitor.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ip/ipmonitor.c b/ip/ipmonitor.c
index baf25a8..4b1d469 100644
--- a/ip/ipmonitor.c
+++ b/ip/ipmonitor.c
@@ -141,8 +141,8 @@ int do_ipmonitor(int argc, char **argv)
 			lprefix=1;
 			groups = 0;
 		} else if (matches(*argv, "neigh") == 0) {
-			lneigh=1;
-			groups = 0;		
+			lneigh = 1;
+			groups = 0;
 		} else if (strcmp(*argv, "all") == 0) {
 			groups = ~RTMGRP_TC;
 			prefix_banner=1;
-- 
1.6.3.3

